pyproject.toml

```diff
@@ -50,11 +50,12 @@ dev = [
   "numpydoc",
 
   # For schema generation.
-  "pydantic>=2.6",
+  "pydantic<3",
+  "datamodel-code-generator",
 ]
 
 [project.scripts]
-regenerate-schema = "event_model.documents.generate.__main__:main"
+regenerate-documents = "event_model.generate.__main__:generate"
```

Comment:
I wonder if we should be exposing this as a script, as it's only used internally to the module. Could we ditch this section and call this via `python -m event_model.generate` instead?